doc/rst/source/greenspline.rst

```diff
@@ -95,7 +117,7 @@ Optional Arguments
     or 3-D data: **3**: records contain **x**, *direction(s)*, *v*
     (*direction(s)* in degrees are measured counter-clockwise from the
     horizontal (and for 3-D the vertical axis). **4**: records contain
```

Comment:
Is the *gradient* in options 0,1, and 2 the same as the gradient magnitude *v*? If so, I think all should use the same notation.

Reply:
Fixed